### PR TITLE
Fixes to TRACING.md

### PR DESCRIPTION
**Pull request by pugnascotia**
Just a few documentation fixes. In particularly, the Markdown references were broken.

### CODE REVIEW

**Comment by pugnascotia** [comment]
FYI @philippkahr

**Review by philippkahr — APPROVED** [review]

**Comment by philippkahr** [comment]
LGTM

File: TRACING.md [hunk 1]
```diff
@@ -21,7 +20,7 @@ impossible.
 ## How is tracing configured?
 
 You must supply configuration and credentials for the APM server (see below).
-In your `elasticsearch.yml` add the following configuration:
+In your `elasticsearch.yml` add the following configuration. Note that the
```
Comment (ChrisHegarty) [inline TRACING.md:23]: Is this a mistake?

**Review by ChrisHegarty — APPROVED** [review]

File: TRACING.md (hunk 1 above)
Comment (pugnascotia) [reply TRACING.md]: Argh, yes, I was going to expand a point but then...didn't.

**Pull request merged**